syntax = "proto3";

message MxAds1015Store {
  repeated int32 readings = 1;
}
